src/content/about/using-cache.md

```diff
@@ -3,143 +3,39 @@ title: Using the Cache
 weight: 3
 ---
 
-Workers provide a way to customize cache behavior using Cloudflare's classic CDN as well as a private cache space. To learn about the benefits of caching see our Learning Center's article on [What is Caching?](https://www.cloudflare.com/learning/cdn/what-is-caching/)
+Workers provide a way to customize cache behavior using Cloudflare's CDN for a zone’s private cache space. To learn about the benefits of caching see our Learning Center's article on [What is Caching?](https://www.cloudflare.com/learning/cdn/what-is-caching/)
 
 ## Interacting with the Cloudflare Cache
 
+Cache is globally distributed and namespaced by zone of the incoming request, meaning a Worker script can not access or set a cache resource on an orange clouded zone’s namespace that differs from the zone of the incoming request.
+
 Conceptually, there are two ways to interact with Cloudflare's Cache using a Worker:
 
-- Store responses on Cloudflare's classic CDN, which can be done through calling `fetch()` in a Workers script. Custom cache behavior via a Worker include:
-  - Setting Cloudflare cache rules (i.e. operating on the cf object)
-  - Setting custom cache headers (i.e. Cache-control). This can impact browser as well as Cloudflare Cache behavior.
-- Store responses on a zone scoped namespace separate from Cloudflare's traditional cache using the Cache API from a Workers script. One can control cache behavior of even assets not proxied on Cloudflare.
+- Call to `fetch()` in a Workers script. Requests proxied through Cloudflare are cached even without Workers according to a default or zone’s configured behavior (e.g. static assets like files ending in .jpg are cached by default). Workers can further customize this behavior by:
+  - Setting Cloudflare cache rules (i.e. operating on the `cf` object of a [request](/reference/apis/request/).
+  - Setting custom cache headers (i.e. `Cache-control`). This can impact browsers as well as Cloudflare Cache behavior.
```

Comment:
This bullet doesn't quite work: our cache actually ignores `Cache-Control` headers on requests. It's only on responses from the origin, or responses put via the Cache API, where `Cache-Control` is honored. Also, the ability to influence browsers' caches is orthogonal to whether the response came from `fetch()` or the Cache API, so might be a bit misleading to state here.

This bullet is worth putting under the Cache API bullet, without mentioning the browser, because it's an example of assets "that would otherwise rely on headers from an origin."

Reply:
```suggestion
```
Thanks for pointing that out. I totally agree that comes off wrong. What do you think of splitting this into two paragraphs? like one for Cloudflare's cache (i.e. modifying request headers/settings to Cloudflare's cache) and one for setting browser cache control (i.e. headers on a response) what do you think is the best way to describe that? I think this part is important and would love to see if you come up with something better.

Reply:
I think talking about browser cache control in a separate paragraph, sibling to the discussion of Cloudflare's cache, would make sense. I think the main points we need to make clear are: you can control either cache with a Cache-Control response header; if you send the response to Cloudflare's cache with `cache.put()`, the header controls Cloudflare's cache; if you send the response to the eyeball with `event.respondWith()`, the header controls the browser's cache.

One question I don't know the answer to is whether the Cache-Control header is returned on a response from `cache.match()` or `fetch()` (i.e., if I `cache.put()` something with a custom header, do I have to worry about that header leaking to the browser?). I imagine it might depend on what page rules are set.

Reply:
OK I changed the wording here let me know what you think @harrishancock